python/pip_install/pip_repository.bzl

```
@@ -719,28 +726,14 @@ def _whl_library_impl(rctx):
     # Manually construct the PYTHONPATH since we cannot use the toolchain here
     environment = _create_repository_execution_environment(rctx, python_interpreter)
 
-    result = rctx.execute(
-        args,
+    repo_utils.execute_checked(
+        rctx,
+        op = "whl_library.ResolveRequirement({}, {})".format(rctx.attr.name, rctx.attr.requirement),
```

nit: we could also have this semantics for the op, where the first item is the function name and the remaining are parameters which will be formated to: `op[0] + "(" + ",".join(op[1:]) + ")"`
```suggestion
        op = ["whl_library.ResolveRequirement", rctx.attr.name, rctx.attr.requirement],
```

---

I like the idea. It also gave me another idea: automatically add the "whl_library" part (the repo rule name) by looking for a `_repo_name` attribute we set on all our repo rules.

But, I'm strapped for time today, so I'm going to merge this as-is.